projectRouter.js

```diff
@@ -0,0 +1,76 @@
+const express = require("express");
+const Project = require("./data/helpers/projectModel");
+const Actions = require("./data/helpers/actionModel");
+
+const projectRouter = express.Router();
+
+// Get all
+
+projectRouter.get('/', async(req, res) => {
+    try{
+        const project = await Project.get();
+        res.status(200).json({project});
+    } catch( error ) {
+        res.status(500).json({message: "Error retrieving projects"});
+    }
+});
+
+//Get by Id
+
+projectRouter.get('/:id', async (req,res) => {
+    try{
+        const project = await Project.get(req.params.id);
+        res.status(200).json({project});
+    } catch(error){
+        res.status(500).json({ message: "Error retrieving Project"});
+    }
+});
+
+//Get actions by project Id
+
+projectRouter.get('/actions/:id', async (req,res) => {
+    try{
+        const project = await Project.getProjectActions(req.params.id);
+        res.status(200).json({project});
```

Comment:
add in a conditional here of if(!project) res.status(404),json({message: "couldnt find project/actions"}) or something along those lines